devnet-sdk/testing/testlib/validators/lowlevel.go

```diff
@@ -54,7 +54,8 @@ func lowLevelSystemValidator(sysMarker interface{}) systest.PreconditionValidato
 }
 
 func AcquireLowLevelSystem() (LowLevelSystemGetter, systest.PreconditionValidator) {
-	sysMarker := &struct{}{}
+	sysMarker := newSentinelMarker()
```

Comment:
I think you should just be able to use `new(byte)` instead of `&struct{}{}` instead of keeping a global counter or?

I made that fix and validated it by a run in the external `devnet-sdk` testing repo, see description in the PR https://github.com/ethereum-optimism/optimism/pull/14514